drivers/node/node.go

```diff
@@ -96,7 +96,7 @@ type Driver interface {
 
 	// YankDrive simulates a failure on the provided drive on the given node.
 	// It returns the UUID of the drive which can be used to recover it back
```

Comment:
nit: add a doc describing the 2nd argument being returned here.

Reply:
If the UUID is not being used anymore we can get rid of it and just return the busID.